ansible/roles/ocp4-workload-logging/files/eo_namespace.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat 
```

Comment:
I know the docs say this, but I think we have been putting this in openshift-operators (where it lands using UI to install as well), which is default namespace for cluster wide. I don't know if that is a docs bug or what, but maybe we could standardize on the "right" namespace.

Reply:
I'm pretty sure that would make managing the operator group a nightmare.

Reply:
I don't think so...there is already an operatorgroup in place for the openshift-operators namespace that handles this for global/cluster wide operators

```
[nstephan@bastion ~]$ oc get operatorgroup -A
NAMESPACE                              NAME                           AGE
openshift-monitoring                   openshift-cluster-monitoring   133m
openshift-operator-lifecycle-manager   olm-operators                  133m
openshift-operators                    global-operators               133m
[nstephan@bastion ~]$ oc get operatorgroup global-operators -n openshift-operators
NAME               AGE
global-operators   133m
[nstephan@bastion ~]$ oc get operatorgroup global-operators -n openshift-operators -oyaml
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  creationTimestamp: "2020-03-09T18:56:52Z"
  generation: 1
  name: global-operators
  namespace: openshift-operators
  resourceVersion: "4965"
  selfLink: /apis/operators.coreos.com/v1/namespaces/openshift-operators/operatorgroups/global-operators
  uid: bc9f3eb5-b6c6-4244-a845-5e23f91b7fff
status:
  lastUpdated: "2020-03-09T19:02:01Z"
  namespaces:
  - ""
```

Reply:
I'll double check with the engineering lead. Depending on what he says I'll update the workload once again - or not. :-)